js/mocha-tests/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "mocha-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --ignore-path .gitignore .",
+    "test": "mocha --compilers js:babel-core/register --require test/test-setup.js --require babel-polyfill test/*.test.js",
+    "test:watch": "npm run test -- --watch"
+  },
+  "author": "superzadeh <superzadeh@outlook.com>",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-core": "6.22.1",
+    "babel-eslint": "7.1.1",
+    "babel-plugin-rewire": "1.0.0-rc-4",
+    "babel-polyfill": "6.22.0",
+    "babel-preset-es2015": "6.22.0",
+    "babel-preset-stage-0": "6.22.0",
+    "chai": "3.5.0",
+    "eslint": "3.15.0",
+    "eslint-config-airbnb-base": "11.1.0",
+    "eslint-plugin-babel": "4.0.1",
+    "eslint-plugin-chai-expect": "1.1.1",
+    "eslint-plugin-import": "2.2.0",
+    "eslint-plugin-mocha": "4.8.0",
+    "mocha": "3.2.0",
+    "sinon": "1.17.7",
+    "sinon-chai": "2.8.0"
+  },
+  "babel": {
```

Comment:
the `.babelrc` at the root should be removed, my bad. There's no differences in using `package.json` or `.babelrc`. Would you rather keep all settings in `.babelrc` or `package.json`?